features/unit/sourcemap.feature

```diff
@@ -0,0 +1,11 @@
+@unit
+Feature: SourceMap
+
+    @unit.sourcemap.pctoline
+    Scenario Outline: Parse a SourceMap from json and compare with expected pc:line string
+        Given a source map json file "<sourcemap-file>"
+        Then the string composed of pc:line number equals "<pc-to-line>"
```

Comment:
Looks good, just for extra peace of mind, I'd test the line:pc mapping as well, since it is exposed to the public.